ggplot/ggplot.py

```diff
@@ -122,417 +84,389 @@ class _empty(object):
             pass
         result = _empty()
         result.__class__ = self.__class__
+        # don't make a deepcopy of data, or plot_env
+        shallow = {'data', 'plot_env'}
         for key, item in self.__dict__.items():
-            # don't make a deepcopy of data!
-            if key == "data":
+            if key in shallow:
                 result.__dict__[key] = self.__dict__[key]
                 continue
             result.__dict__[key] = deepcopy(self.__dict__[key], memo)
 
         return result
 
-    def draw(self):
```

Comment:
I used draw in some SO questions. It would be nice if that could at least be kept with a warning and call the function which now does the job (render?)